versions.sh

```diff
@@ -105,8 +105,8 @@ for version in "${versions[@]}"; do
 
 	rpmRepo="https://repo.mysql.com/yum/mysql-$version-community/docker/el/$oracleVersion"
 	case "$version" in
-		innovation) toolsRepo="https://repo.mysql.com/yum/mysql-tools-innovation-community/el/$oracleVersion" ;;
```

Comment:
Tnere is still an innovation section the the versions files. Shouldn't both lines be added?

Reply:
In our current code in this section, "innovation" is the special case, but now we've got a new example that matches innovation, so the change here is swapping 8.0 to be the special case going forward instead (now that 5.x is EOL, that seems appropriate). :+1:

Reply:
For `innovation` that is the value of `$version`, so the new case should work for both.